docs/en/reference/partial-objects.rst

```diff
@@ -0,0 +1,88 @@
+Partial Objects
+===============
+
+A partial object is an object whose state is not fully initialized
+after being reconstituted from the database and that is
+disconnected from the rest of its data. The following section will
+describe why partial objects are problematic and what the approach
+of Doctrine2 to this problem is.
```

Comment:
Should we keep using Doctrine2 here? I'm not sure our audience knows what Doctrine1 refers to.

Reply:
We already had PRs where Doctrine2 was replaced with Doctrine. Let's do the same here.